Decred Wallet/AppDelegate.swift

```diff
@@ -182,15 +182,16 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
     
     func handlerNotification(response: UNNotificationResponse) {
         guard let data = response.notification.request.content.userInfo as? [String: String] else { return }
-        if let navigation = UIApplication.shared.keyWindow?.rootViewController as? UINavigationController, navigation.viewControllers.count > 0 {
-            let storyboard = UIStoryboard(name: "Politeia", bundle: nil)
-            if let politeiaVC = storyboard.instantiateViewController(withIdentifier: "PoliteiaDetailController") as? PoliteiaDetailController {
-                politeiaVC.isNotificationOpen = true
-                politeiaVC.proposalId = data["proposalId"]
-                navigation.pushViewController(politeiaVC, animated: true)
+        if (data["proposalId"] != nil) {
```

Comment:
`proposalId` is used in multiple places, consider making it a constant.

https://github.com/planetdecred/dcrios/blob/master/Decred%20Wallet/Utils/NotificationsManager.swift#L83